ansible/playbook.yaml

```diff
@@ -66,11 +66,17 @@
     - name: "secret_key"
```

Comment:
Worth noting there's no requirement to keep these playbooks up to date going forward - in a real project we'd certainly want to but since we're no longer using them and we can't test them, it's a good sign that really we'd be better off deleting them rather than assuming we've correctly updated them